packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1205,7 +1214,7 @@ export class SnapController extends BaseController<
    * stopped.
    */
   public async stopSnap(
-    snapId: SnapId,
+    snapId: ValidatedSnapId,
```

Comment:
For public functions should we take any snapId and assert its validity? Assuming it to be valid might be unsafe?